LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -120,6 +123,20 @@ extension TherapySettingsView {
 // MARK: Sections
 extension TherapySettingsView {
     
+    private var prescriptionSection: some View {
+        SectionWithEdit(isEditing: .constant(false),
+                        addExtraSpaceAboveSection: true,
+                        title: LocalizedString("Prescription", comment: "title for prescription section"),
+                        descriptiveText: prescriptionDescriptiveText,
+                        destination: EmptyView(), content: { EmptyView() })
+    }
+    
+    private var prescriptionDescriptiveText: String {
+        String(format: LocalizedString("Submitted by %@, %@", comment: "Format for prescription descriptive text (1: providerName, 2: datePrescribed)"),
```

Comment:
[nit] You might consider numbering the format string arguments (i.e. `%1$@` and `%2$@`) to facilitate reordering for translators